c7n/resources/swf.py

```diff
@@ -0,0 +1,65 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.utils import local_session
+from c7n import tags
+
+
+@resources.register('swf')
+class SimpleWorkflowDomain(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = arn_type = 'swf'
+        enum_spec = ('list_domains', 'domainInfos', {'registrationStatus': 'REGISTERED'})
+        id = name = 'name'
+        universal_taggable = object()
+        permission_augment = ('swf:ListTagsForResource',)
+
+    def augment(self, rules):
+        client = local_session(self.session_factory).client('swf')
+
+        def _add_tags(r):
+            r['Tags'] = client.list_tags_for_resource(
```

Comment:
should use universal augment from tags since this resource is supported by resource group tagging.

Reply:
Fixed this